front/components/assistant/AssistantDetailsDropdownMenu.tsx

```diff
@@ -140,50 +139,58 @@ export function AssistantEditionMenu({
       <DropdownMenu className="text-element-700">
         <DropdownMenu.Button>{dropdownButton}</DropdownMenu.Button>
         <DropdownMenu.Items width={220}>
-          {showAddRemoveToList && (
-            <DropdownMenu.SectionHeader label="Edition" />
-          )}
-          {/* Should use the router to have a better navigation experience */}
-          {(isBuilder(owner) || !isAgentWorkspace) && (
-            <DropdownMenu.Item
-              label="Edit"
-              href={`/w/${owner.sId}/builder/assistants/${
-                agentConfiguration.sId
-              }?flow=${
-                isAgentWorkspace
-                  ? "workspace_assistants"
-                  : "personal_assistants"
-              }`}
-              icon={PencilSquareIcon}
-            />
-          )}
           <DropdownMenu.Item
-            label="Duplicate (New)"
-            href={`/w/${owner.sId}/builder/assistants/new?flow=personal_assistants&duplicate=${agentConfiguration.sId}`}
-            icon={ClipboardIcon}
+            label="Start conversation"
+            href={`/w/${owner.sId}/assistant/new?mention=${agentConfiguration.sId}`}
```

Comment:
Not a big fan on `mention` 🤔 since it represent a agent configuration id.

Reply:
I renamed to assistant.